lampo-vls/src/keys_manager.rs

```diff
@@ -0,0 +1,150 @@
+use lampo_common::vls::proxy::vls_protocol_client::SpendableKeysInterface;
+use lampo_common::vls::proxy::vls_protocol_client::{DynSigner, KeysManagerClient};
+use lampo_common::vls::signer::bitcoin::secp256k1::ecdh::SharedSecret;
+use lampo_common::vls::signer::bitcoin::secp256k1::ecdsa::RecoverableSignature;
+use lampo_common::vls::signer::bitcoin::secp256k1::ecdsa::Signature as EcdsaSignature;
+use lampo_common::vls::signer::bitcoin::secp256k1::schnorr::Signature;
+use lampo_common::vls::signer::bitcoin::secp256k1::{All, Secp256k1};
+use lampo_common::vls::signer::bitcoin::secp256k1::{PublicKey, Scalar};
+use lampo_common::vls::signer::bitcoin::{bech32::u5, Address, Script};
+use lampo_common::vls::signer::bitcoin::{Transaction, TxOut, Witness};
+use lampo_common::vls::signer::invoice::bolt12::UnsignedBolt12Invoice;
+use lampo_common::vls::signer::lightning::ln::{msgs, script::ShutdownScript};
+use lampo_common::vls::signer::lightning::offers::invoice_request::UnsignedInvoiceRequest;
+use lampo_common::vls::signer::lightning::sign::SpendableOutputDescriptor;
+use lampo_common::vls::signer::lightning::sign::{EntropySource, NodeSigner, SignerProvider};
+use lampo_common::vls::signer::lightning::sign::{KeyMaterial, Recipient};
+
+use crate::util::create_spending_transaction;
+
+#[allow(dead_code)]
+/// Holds an instance of KeysManagerClient which interacts with the VLS protocol to fetch or manage keys.
+// FIXME: This should be implemented under VLS?
+pub struct LampoKeysManager {
+    /// The KeysManagerClient is a client-side interface for interacting with the key management functionalities of the VLS signer.
+    client: KeysManagerClient,
+    sweep_address: Address,
+}
+
+impl LampoKeysManager {
+    pub fn new(client: KeysManagerClient, sweep_address: Address) -> Self {
+        LampoKeysManager {
+            client,
+            sweep_address,
+        }
+    }
+}
```

Comment:
This is already implemented inside the lampo_common, what it is stopping you to use it?